particles/Tutorial/Javascript/Demo/Game.arcs

```diff
@@ -0,0 +1,60 @@
+// Tutorial Demo
```

Comment:
I don't fully understand why Game.arcs and TicTacToe.arcs are separate files. Is the idea that particles and recipes in Game.arcs are universal to all games? Or at least to a class of games?